### PR TITLE
Add node.js fundamentals study link

### PR DESCRIPTION
**Pull request by nazreenpe**
Added a node.js fundamentals study link

### CODE REVIEW

File: node-js/node-js.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ This lab exercise will introduce Node.js, a powerful tool that lets you run Java
 
 ### Materials
 
+- [The beginner's guide: Understanding Node.js & Express.js fundamentals](https://medium.com/@LindaVivah/the-beginners-guide-understanding-node-js-express-js-fundamentals-e15493462be1)
```
Comment (alodahl) [inline node-js/node-js.md]: Hi Nazreen, how long does this take? Can you add the time next to it, and add that to the the Projected Time?

**Comment by alodahl** [comment]
Hi Nazreen, can you make the requested changes so I can merge?

**Comment by nazreenpe** [comment]
Sure.
I will do that.

Thank you

**Comment by alodahl** [comment]
@nazreenpe sending you a reminder about his one.

**Pull request merged**